webrtc-stats.html

```diff
@@ -515,6 +516,15 @@ <h3>
               accessed by the <code><a>RTCRemoteOutboundRtpStreamStats</a></code>.
             </p>
           </dd>
+          <dt>
+            <dfn><code>media-source</code></dfn>
+          </dt>
+          <dd>
+            <p>
+              Statistics for the media produced by a local <code>MediaStreamTrack</code> that is
+              currently attached to an <code>RTCRtpSender</code>.
+            </p>
```

Comment:
Add what the type of the dictionary is - "it is accessed by the <code>DictionaryName</code>" is the standard incantation above, even though the words don't make much sense.

Reply:
Done